config/software/libxslt.rb

```diff
@@ -41,7 +41,7 @@
   env = with_standard_compiler_flags(with_embedded_path({}, msys: true), bfd_flags: true)
 
   patch source: "libxslt-cve-2015-7995.patch", env: env
-  patch source: "libxslt-solaris-configure.patch", env: env if solaris?
+  patch source: "libxslt-solaris-configure.patch", env: env if solaris2?
```

Comment:
If possible let's use `solaris?` (which is an easier to understand alias to `solaris2?`). Take a look at:
https://github.com/sethvargo/chef-sugar/blob/b9b3b643c27a0828f56d7355bb4d05577bd90a38/lib/chef/sugar/platform.rb#L198-L208